engine/src/main/java/com/sandboni/core/engine/finder/JarFinder.java

```diff
@@ -69,7 +69,7 @@ private String getFileContent(JarFile jar, JarEntry entry) {
             InputStream inputStream = jar.getInputStream(entry);
             return IOUtils.toString(inputStream, Charset.defaultCharset());
         } catch (IOException e) {
-            log.error("Error during reading file: " + entry.getName(), e);
+            log.error(String.format("Error during reading file: %s", entry.getName()), e);
```

Comment:
think Sonar won't like it -> use {} instead of String.format &  %s,

Reply:
I changed this because sonar failed on old code..

Reply:
but then it will not handle the exception (there is no signature for string, arg, arg,.., throwable)

Reply:
I've rolled this back so you can have a look